### ISSUE TITLE
[A11y][ARIA]The purpose of the Learn more link is not descriptive

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8184]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1184490
## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450)

## Repro Steps:

    Hit the URL and login with appropriate credentials
    Tab till "upload tab" item and press enter
    Tab till "Verify" drop down and press enter.
    Tab till "Learn more" link
    Check whether the purpose of the link is defined by the name of the link or not

## Actual Results:

The purpose of the link is not descriptive for the "Learn more" and "Read more" links

   
## Expected Results:
The purpose of the link should be more descriptive. Screen reader should understand it clearly
## User Impact:
Users with visual disabilities may find it difficult to understand the purpose of the link, if link name is not descriptive.

## Notes
Updating the aria-label in this case to a more descriptive text is sufficient. See https://github.com/NuGet/NuGetGallery/issues/7879 for a similar fix.

**Comment by ryuyu** [comment]
Reopening due to issue reactiviation. Looks like a few more spots need updated decriptions. https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1184490

See the attachments in the linked issue for details.
